### PR TITLE
Fix duration.cpp lint error

### PR DESCRIPTION
**Pull request by dabonnie**
Fix linting error found via the [nightly build](https://ci.ros2.org/view/nightly/job/nightly_linux_debug/1388/testReport/junit/rclcpp/uncrustify/src_rclcpp_duration_cpp/) test results.

### CODE REVIEW

**Comment by dabonnie** [comment]
@wjwwood @ivanpauno could either of you please review? :-)

File: rclcpp/src/rclcpp/duration.cpp [hunk 1]
```diff
@@ -183,7 +183,8 @@ bounds_check_duration_scale(int64_t dns, double scale, uint64_t max)
   auto abs_dns = static_cast<uint64_t>(std::abs(dns));
   auto abs_scale = std::abs(scale);
   if (abs_scale > 1.0 &&
-    abs_dns > static_cast<uint64_t>(static_cast<long double>(max) / static_cast<long double>(abs_scale)))
+    abs_dns >
```
Comment (dirk-thomas) [inline rclcpp/src/rclcpp/duration.cpp]: How about moving `abs_dns >` to the line above?

Comment (dabonnie) [reply rclcpp/src/rclcpp/duration.cpp]: > How about moving `abs_dns >` to the line above?

Sure, I don't have a preference other than silencing the linter failure ;-)

Comment (dirk-thomas) [reply rclcpp/src/rclcpp/duration.cpp]: Great, please update this PR accordingly.

Comment (dabonnie) [reply rclcpp/src/rclcpp/duration.cpp]: > Great, please update this PR accordingly.

Done!

**Comment by dirk-thomas** [comment]
Thanks for the patch.

**Pull request merged**